src/named.rs

```diff
@@ -404,7 +420,14 @@ impl NamedChain {
         use NamedChain as C;
 
         Some(Duration::from_millis(match self {
-            C::Mainnet | C::Taiko | C::TaikoHekla => 12_000,
+            C::Mainnet
+            | C::Taiko
+            | C::TaikoHekla
+            | C::Mandala
+            | C::KaruraTestnet
+            | C::AcalaTestnet
+            | C::Karura
+            | C::Acala => 12_000,
```

Comment:
If possible please make sure to use alphabetical ordering for lists like this

Reply:
I only sorted some of the matching items. It doesn't feel good to order everything I changed.